shared/src/main/scala/amf/core/client/common/validation/ProfileNames.scala

```diff
@@ -64,6 +73,7 @@ object ProfileName {
       case Raml08Profile.p  => Some(Raml08Profile)
       case AsyncProfile.p   => Some(AsyncProfile)
       case Async20Profile.p => Some(Async20Profile)
+      case GrpcProfile.p    => Some(GrpcProfile)
```

Comment:
Missing Proto3 here and below?

Reply:
We deleted Proto3 as were redundant with Grpc. Do you thing that we need that as a separated vendor? it use to invoke the same plugin that grpc